### PR TITLE
Add running doctest to pytest default

### PR DESCRIPTION
**Pull request by Cjkjvfnby**
### Describe your change:
Add default arguments to pytest.ini.  This helps when you want to test your code locally.

```
  -l, --showlocals      Show locals in tracebacks (disabled by default)
  --doctest-modules     Run doctests in all .py modules
  --doctest-continue-on-failure
                        For a given doctest, continue to run after the first failure

```

* [ ] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [x] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [ ] This pull request is all my own work -- I have not plagiarized.
* [ ] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [ ] All new Python files are placed inside an existing directory.
* [ ] All filenames are in all lowercase characters with no spaces or dashes.
* [ ] All functions and variable names follow Python naming conventions.
* [ ] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [ ] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms include at least one URL that points to Wikipedia or another similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by cclauss** [comment]
Can you please run https://pypi.org/project/ini2toml on this file and also `[.coveragerc](https://github.com/TheAlgorithms/Python/blob/master/.coveragerc)` and then put the results in a new `pyproject.toml` file so that we can move into the [PEP 621](https://www.python.org/dev/peps/pep-0621/) future?

File: pytest.ini [hunk 1]
```diff
@@ -1,5 +1,4 @@
-# Setup for pytest
 [pytest]
 markers =
     mat_ops: mark a test as utilizing matrix operations.
-addopts = --durations=10
+addopts = --durations=10 --doctest-modules --doctest-continue-on-failure --showlocals
```
Comment (cclauss) [inline pytest.ini:4]: Let's leave out continue on failure.

Comment (Cjkjvfnby) [reply pytest.ini:4]: What is the story behind that?  

Usually, it's great to see all problems at once.

**Comment by cclauss** [comment]
I question the utility of coverage -- It generates a TON of output on every test run and no one reads it.  If folks want this, they can run it on their own computers and time.

**Comment by Cjkjvfnby** [comment]
> I question the utility of coverage -- It generates a TON of output on every test run and no one reads it.

Totally agree, I just noticed that it exists by accident.   If tests pass I won't go to see action logs.
I don't like that text format and prefer to use HTML reporting on my machine. 

Maybe some aggregator similar to pre-commit.ci could help with it. I have never used such tools. https://about.codecov.io/pricing/ 

There are a bunch of GitHub actions that work with coverage. Need to do a bit of experimenting with that https://github.com/marketplace?type=actions&query=python+coverage+

**Review by cclauss — CHANGES_REQUESTED** [review]

**Comment by Cjkjvfnby** [comment]
This is the same report but a bit more detailed.

```
--doctest-continue-on-failure
                        For a given doctest, continue to run after the first failure
```


```python
def foo():
    """
    int(1)
    >>> 2

    int(2)
    >>> 3
    """
```

`pytest --doctest-modules scratch_7.py`
```
================================================================== FAILURES ==================================================================
__________________________________________________________ [doctest] scratch_7.foo ___________________________________________________________
031 
032     int(1)
033     >>> 2
Expected nothing
Got:
    2

scratch_7.py:33: DocTestFailure
========================================================== short test summary info =========================================================== 
```

`pytest --doctest-modules --doctest-continue-on-failure scratch_7.py`
```
================================================================== FAILURES ================================================================== 
__________________________________________________________ [doctest] scratch_7.foo ___________________________________________________________ 
031
032     int(1)
033     >>> 2
Expected nothing
Got:
    2

scratch_7.py:33: DocTestFailure
031
032     int(1)
033     >>> 2
034
035     int(2)
036     >>> 3
Expected nothing
Got:
    3

scratch_7.py:36: DocTestFailure
========================================================== short test summary info ===========================================================
```

**Comment by Cjkjvfnby** [comment]
i have moved pytest and coverage setting to pyproject.toml and return `--doctest-continue-on-failure` back (https://github.com/TheAlgorithms/Python/pull/7840#issuecomment-1295810196)

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+[tool.pytest.ini_options]
+markers = [
+    "mat_ops: mark a test as utilizing matrix operations.",
+]
+addopts = "--durations=10 --doctest-modules --doctest-continue-on-failure --showlocals"
```
Comment (dhruvmanila) [inline pyproject.toml:5]: Let's have a list of strings instead if that's possible.

File: pyproject.toml [hunk 3]
```diff
@@ -2,8 +2,12 @@
 markers = [
     "mat_ops: mark a test as utilizing matrix operations.",
 ]
-addopts = "--durations=10 --doctest-modules --doctest-continue-on-failure --showlocals"
-
+addopts = [
+    "--durations=10",
+    "--doctest-modules",
+    "--doctest-continue-on-failure",
```
Comment (cclauss) [inline pyproject.toml:8]:
```suggestion
```

**Review by cclauss — APPROVED** [review]
Nice!

**Pull request merged**